index.html

```diff
@@ -6,11 +6,49 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Comforter&family=Estonia&family=Praise&display=swap');
+        </style>
 </head>
 
-<body>
-    <!-- Add your markup here -->
+<body class="container">
+    <div></div>
+    <header class="header">
+            <div class="banner-text">
+                <h1>B.G. CAKES</h1>
+            </div>
+    </header>
+    <main class="welcome">
+        <div class="main">
+            <div class="main-text">
+                <h2>Welcome</h2>       
+                <p>Welcome to the most BADMAN cake delivery service that you have EVER
+                    encountered. <br> <br>Our cakes are SO GANGSTER, that you won't even want to eat them!
+                </p>
+            </div>
+            <div>    
+                <img src="./img/syed-maaz-QLSh4y2v93c-unsplash.jpg" alt="Designer Cake" class="main-img"/>      
+            </div>
+        </div>
+    </main>
+    <section class="gallery">
```

Comment:
`<main>` mostly should be around the main part of the page except footer and header. The `section` should be in the `<main>`.